src/cpp/src/utils.cpp

```diff
@@ -266,6 +266,42 @@ ov::Core singleton_core() {
     return core;
 }
 
+void print_compiled_model_properties(ov::CompiledModel& compiled_Model, ov::Core& core) {
+    // Specify the name of the environment variable
+    const char* env_var_name = "OPENVINO_LOG_LEVEL";
```

Comment:
```suggestion
    const char* env_var_name = "OPENVINO_GENAI_LOG_LEVEL";
```
Splitting the ov and genai env vars would allow better configuration granularity

Reply:
When setting the environment variable OPENVINO_LOG_LEVEL, the optimal-intel API will also print the properties of the compiled model, so the genai API shares the same environment variable.
ov::log::Level::WARNING can be found in
https://github.com/openvinotoolkit/openvino/blob/b2bfd851a1568c7cb496780eec9af092e0935a1c/src/inference/include/openvino/runtime/properties.hpp#L611